packages/experimental/src/experimental-list/task-item.scss

```diff
@@ -65,13 +65,50 @@ $task-alert-yellow: #f0b849;
 	}
 
 	.woocommerce-task-list__item-content {
+		max-height: 0;
+		opacity: 0;
 		margin-top: $gap-smallest;
-		margin-bottom: $gap-smallest;
+		overflow: hidden;
+
+		&.woocommerce-task-list__item-content-enter {
+			opacity: 0;
+			max-height: 0;
+		}
+
+		&.woocommerce-task-list__item-content-enter-active {
+			opacity: 1;
+			max-height: 100vh;
```

Comment:
Do we want to use a smaller value here, like 150px maybe? Currently there is a bit of a delay between the one being opened and the one being closed. A smaller value makes it a bit more seamless.

I am starting to think that we might want a custom CSSTransition component that automatically add's the specific height to a component so we could always use it for this type of transition.

Reply:
> Do we want to use a smaller value here, like 150px maybe?

There isn't any restriction on the content for each task item, so I didn't want to use a fixed height. @elizaan36 was cool with the slight delay last I checked (but keep me honest!)

> I am starting to think that we might want a custom CSSTransition component that automatically add's the specific height to a component so we could always use it for this type of transition.

Ooh, that would be a great cooldown task - and I think the right way to tackle the delay.

Reply:
> There isn't any restriction on the content for each task item, so I didn't want to use a fixed height. @elizaan36 was cool with the slight delay last I checked (but keep me honest!)

Gotcha, I will approve this in the mean time then, given it's a minimal change if we do decide to change it.

Reply:
Good idea, @louwie17 ! I'm fine with the transition either way but the opening and closing could happen more simultaneously. And you're right that it would feel more seamless.